File: PYTHON/Practica8/PROBLEMA_DIAMANTE/B.py
from C import C

class B(C):
    def __init__(self, y, z):
        C.__init__(self, z)

        self.y = y
    def incrementaYZ(self):
        self.y += 1
        self.z += 1
    def incrementaZ(self):
        self.z += 1

File: JAVA/Practica8/PROBLEMA_DIAMANTE/B.java
package JAVA.Practica8.PROBLEMA_DIAMANTE;

public class B extends C {
    public int y;
    public B(int y, int z) {
        super(z);
        this.y = y;
    }
    public void incrementaYZ() {
        this.y += 1;
        this.z += 1;
    }
    public void incrementaZ() {
        this.z += 1;
    }
}